pallets/rc-migrator/src/proxy.rs

```diff
@@ -213,3 +215,16 @@ impl<T: Config> PalletMigration for ProxyAnnouncementMigrator<T> {
 		}
 	}
 }
+
+impl<T: Config> RcMigrationCheck for ProxyProxiesMigrator<T> {
+	type RcPrePayload = usize; // number of delegators
+
+	fn pre_check() -> Self::RcPrePayload {
+		pallet_proxy::Proxies::<T>::iter_keys().count()
+	}
+
+	fn post_check(_: Self::RcPrePayload) {
+		let count = pallet_proxy::Proxies::<T>::iter_keys().count();
+		assert_eq!(count, 0, "All proxies are removed from the relay");
```

Comment:
Somehow it works here, so maybe the other migration was faulty where it was missing items 🤷‍♂️